### PR TITLE
BaSP-TG-143: Implement and update form Projects

### PR DESCRIPTION
**Pull request by MartinDidoli**
- Forms updated with read-hook-form
- Validations with joi added
- Absolute routes added
- Figma Styles corrected: Forms have 2 columns
- Confirm button is on the left and cancel on the right

### CODE REVIEW

**Review by Juanoauil — COMMENTED** [review]
nice work

**Review by dayanagm96 — APPROVED** [review]

**Review by Juanoauil — APPROVED** [review]

**Review by marcomigliaro — APPROVED** [review]

File: src/Components/Projects/AddNew.js [hunk 1]
```diff
@@ -45,280 +100,134 @@ const AddNew = () => {
...
-            inputValue={project.admins}
-            placeholder="enter an admin here"
-            warningMsg="Please check the information"
-            handleInput={handleInputAdmin}
-            handleClick={handleClickAdmin}
-            handleBlur={handleBlurAdmin}
-            showWarning={showWarningAdmin}
-          ></Input>
+          <Button type={('submit', styles.modalProjectBtn)}>Confirm</Button>
+          <Button type={styles.backBtn} handleClick={() => (window.location.href = '/projects')}>
```
Comment (SuarezAlexisEzequiel) [inline src/Components/Projects/AddNew.js:204]: Can we use a 'history.push' here?

File: src/Components/Projects/EditProject.js [hunk 2]
```diff
@@ -44,290 +112,146 @@ const EditProject = () => {
...
-            inputValue={project.admins}
-            placeholder="enter an admin here"
-            warningMsg="Please check the information"
-            handleInput={handleInputAdmin}
-            handleClick={handleClickAdmin}
-            handleBlur={handleBlurAdmin}
-            showWarning={showWarningAdmin}
-          ></Input>
+          <Button type={('submit', styles.modalProjectBtn)}>Edit Project</Button>
+          <Button type={styles.backBtn} handleClick={() => (window.location.href = '/projects')}>
```
Comment (SuarezAlexisEzequiel) [inline src/Components/Projects/EditProject.js:228]: Same as the comment on Add New, to the 'window.location.href'

**Review by SuarezAlexisEzequiel — APPROVED** [review]
Except for the comment already left, everything seems OK

**Review by SuarezAlexisEzequiel — APPROVED** [review]

**Review by DeepFreq — APPROVED** [review]

**Review by SuarezAlexisEzequiel — APPROVED** [review]

**Review by Lucas-DeLorenzi — APPROVED** [review]

**Review by JulianDeme — APPROVED** [review]

**Pull request merged**